benchmarks.yml

```
@@ -45,6 +45,8 @@ etanni:
   desc: etanni is an older, extremely simple template-lang format that basically turns your template into an "eval" with a lot of heredocs.
 fannkuchredux:
   desc: fannkuchredux from the Computer Language Benchmarks Game.
+json_parser:
```

```suggestion
rb_json_parser:
```

Just to make it clear from the name this is a Ruby JSON parser.